docs/src/components/InstantSearch.md

```diff
@@ -53,7 +53,7 @@ Name | Type | Default | Description | Required
 `index-name` | String  | | The index to target for the search | yes
 `searchFunction` | `(AlgoliaHelper) => void` | | A hook that will be called each time a search needs to be done, with the [helper](https://community.algolia.com/algoliasearch-helper-js/) as a parameter. It’s your responsibility to call `helper.search()`. This option allows you to avoid doing searches at page load for example. | no
 `stalled-search-delay` | Number | `200`  | Time (in ms) before the search is considered unresponsive. Used to display a loading indicator. | no
-`routing` | Boolean or Object | `false` | Enable the default routing feature by passing `true`. More advanced usage is documented [here](https://community.algolia.com/instantsearch.js/v2/guides/routing.html). | no
+`routing` | Object | | documented [here](/advanced/vue-router-url-sync.html). | no
```

Comment:
it might be worth detailing the shape of Object? in this case `{ history, stateMapping }`